### PR TITLE
Add option to roll up phpcs configuration

### LINKED ISSUE #84: Use the parent project's phpcs ruleset
When rolling the Composer config up into the parent, add a prompt to inherit the parent project's `.phpcs.xml`. Then the plugin's `phpcs.xml` can be simplified to just bring in that ruleset and add the text domain and prefix. Similar to [this].(https://github.com/alleyinteractive/wp-starter-project/blob/main/plugins/wp-starter-plugin/.phpcs.xml).

### PR DESCRIPTION
**Pull request by srtfisher**
Prompts the user if they're rolling up their Composer configuration to also roll up their PHPCS configuration to the parent project. If they opt into it, they will get a modified plugin phpcs configuration. Example:

```xml
<?xml version="1.0"?>
<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="WP Starter Plugin Configuration" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/squizlabs/PHP_CodeSniffer/master/phpcs.xsd">
  <description>PHP_CodeSniffer standard for WP Starter Plugin</description>

  <!-- DO NOT ADD ADDITIONAL RULES TO THIS FILE. Modifications belong in the root-level configuration. -->

  <!-- Include Root Rules -->
  <rule ref="../../.phpcs.xml" />

  <rule ref="WordPress.WP.I18n">
    <properties>
      <!--
      Verify that the text_domain is set to the desired text-domain.
      Multiple valid text domains can be provided as a comma-delimited list.
       -->
      <property name="text_domain" type="array" value="wp-starter-plugin" />
    </properties>
  </rule>

  <rule ref="WordPress.NamingConventions.PrefixAllGlobals">
    <properties>
      <property name="prefixes" type="array" value="wp_starter_plugin,ai_,_ai" />
    </properties>
  </rule>
</ruleset>
```

Resolves #84

### CODE REVIEW

File: configure.php [hunk 1]
```diff
@@ -129,6 +129,38 @@ function remove_project_files() {
...
+      Verify that the text_domain is set to the desired text-domain.
+      Multiple valid text domains can be provided as a comma-delimited list.
+       -->
+      <property name="text_domain" type="array" value="' . $plugin_domain . '" />
+    </properties>
+  </rule>
+
+  <rule ref="WordPress.NamingConventions.PrefixAllGlobals">
+    <properties>
+      <property name="prefixes" type="array" value="' . $plugin_domain . ',ai_,_ai" />
```
Comment (emilyatmobtown) [inline configure.php]: The `ai` prefixes are unique to Alley – and I don't see any globals prefixed with them in the plugin. I think we can safely delete these.

Also, the global prefix is snakecase.

```suggestion
      <property name="prefixes" type="array" value="' . str_replace( '-', '_', $plugin_domain ) . '" />
```

File: configure.php [hunk 2]
```diff
@@ -386,6 +420,30 @@ function delete_files( string|array $paths ) {
...
+			if ( file_exists( __DIR__ . '/phpcs.xml' ) ) {
+				foreach ( $parent_files as $parent_file ) {
+					if ( ! file_exists( $parent_file ) ) {
+						continue;
+					}
+
+					if ( confirm( "Do you want to roll up the phpcs configuration to the parent? (This will change the plugin's phpcs configuration to inherit the parent configuration from {$parent_file}.)" ) ) {
+						rollup_phpcs_to_parent(
+							parent_file: '../../' . basename( $parent_file ),
+							local_file: __DIR__ . '/phpcs.xml',
```
Comment (emilyatmobtown) [inline configure.php]: You may want to consider naming the ruleset `.phpcs.xml` to align with the starter project and starter theme.

File: configure.php [hunk 3]
```diff
@@ -386,6 +420,30 @@ function delete_files( string|array $paths ) {
...
+				$parent_folder . '/.phpcs.xml',
+			];
+
+			if ( file_exists( __DIR__ . '/phpcs.xml' ) ) {
+				foreach ( $parent_files as $parent_file ) {
+					if ( ! file_exists( $parent_file ) ) {
+						continue;
+					}
+
+					if ( confirm( "Do you want to roll up the phpcs configuration to the parent? (This will change the plugin's phpcs configuration to inherit the parent configuration from {$parent_file}.)" ) ) {
```
Comment (emilyatmobtown) [inline configure.php:465]: Nice!

**Review by emilyatmobtown — APPROVED** [review]
🌮 Looks good! A couple suggestions. No blockers.

File: configure.php (hunk 1 above)
Comment (srtfisher) [reply configure.php]: The domain in the plugin is already `create-wordpress-plugin` -- the domain should be kebab case (but it really is the same difference 😆 ) https://developer.wordpress.org/themes/functionality/internationalization/#text-domain

**Pull request merged**